CODEOWNERS

```diff
@@ -1,18 +1,41 @@
 # Code ownership information.
-# See https://help.github.com/articles/about-code-owners/ for details.
+# See
+# https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners
+# for details.
 
+# Order is important; the *last* matching pattern takes precedence.
+
+# Default
+#
 # These owners will be the default owners for everything in the repo. Unless a
-# later match takes precedence, # @trusted-contributors will be requested for
-# review when someone opens a pull request.
+# later match takes precedence, @collectd/trusted-contributors will be
+# requested for review when someone opens a pull request.
+*       @collectd/trusted-contributors
 
-/src/intel_pmu.c	@kwiatrox @sunkuranganath
-/src/intel_rdt.c	@kwiatrox @sunkuranganath
-/src/ipmi.c		@anaudx @rjablonx
-/src/mcelog.c		@kwiatrox @sunkuranganath
-/src/virt.c		@anaudx @rjablonx
+# Per-plugin owners
+#
+# These plugins are owned by subject matter experts and require their review.
+/src/intel_pmu.c	@collectd/intel
+/src/intel_rdt.c	@collectd/intel
+/src/ipmi.c		@collectd/intel
+/src/mcelog.c		@collectd/intel
+/src/virt.c		@collectd/intel
 # TODO(#2926): Add the following owners:
-#/src/redfish.c		@kkepka @mkobyli
+#/src/redfish.c		@collectd/intel
+
+# Core
```

Comment:
What about the CODEOWNERS file? I'm of split mind here, on the one hand it feels like this should be for core-maintainers only but on the other hand it would be nice to allow contributors to modify per-plugin settings, e.g. for new plugins.

Reply:
Personally, I'd keep the CODEOWNERS file for core maintainers only. Ideally, there would be no controversy, if a plugin should be added here or not and for core maintainers, it's a quick approval then.

Reply:
Done.